src/Components/Users/UserProfile.tsx

```diff
@@ -578,6 +591,28 @@ export default function UserProfile() {
                       {userData?.weekly_working_hours || "-"}
                     </dd>
                   </div>
+                  <div
+                    className="my-2 sm:col-span-2"
+                    id="videoconnectlink-profile-details"
+                  >
+                    <dt className="text-sm font-medium leading-5 text-black">
+                      Video Connect Link
+                    </dt>
+                    <dd className="mt-1 break-words text-sm leading-5 text-gray-900">
+                      {userData?.video_connect_link ? (
+                        <a
+                          className="text-blue-500"
+                          href={userData?.video_connect_link}
+                          target="_blank"
+                          rel="noreferrer"
+                        >
+                          {userData?.video_connect_link}
+                        </a>
```

Comment:
Should we also consider tracking how often this is clicked?

Reply:
That would be nice! I believe it'll make more sense to track the clicks (for the 3 connect buttons: video/wp/phone) on the doctor connect feature to understand the impact of this feature. Added this!